testRadius/main.go

```diff
@@ -16,14 +16,15 @@ import (
 func main() {
 	fmt.Printf("test start ... \n")
 	packet := radius.New(radius.CodeAccessRequest, []byte(`secret`))
-	rfc2865.UserName_SetString(packet, "208920100001105")
+	// rfc2865.UserName_SetString(packet, "208920100001105")
+	rfc2865.UserName_SetString(packet, "1234")
 	rfc2865.UserPassword_SetString(packet, "Kurtis")
 	response, err := radius.Exchange(context.Background(), packet, "localhost:1812")
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	log.Println("Code:", response.Code)
+    log.Println("Code:", response.Code)
```

Comment:
Indentation here is probably not an intended change, and will probably not pass `gofmt`